code/src/pages/GameScreen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import "nes.css/css/nes.min.css"
+
+import questions, { generateNextQuestion } from '../reducers/questions'
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+`
+
+const GameScreenMainDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 5% 10% 15% 10%;
+  text-align: center;
+  background-size: cover;
+  background-position: center;
+  height: 450px;
+  max-width: 1000px;
```

Comment:
You have the max-width but you do not have any other parameter which would make it stretch as much as possible, so on the ultrawide screen your page looks like this:
![image](https://user-images.githubusercontent.com/50926159/118015438-8c747800-b354-11eb-9cd8-3a0a0c27c4c5.png)